app/src/cc/arduino/contributions/packages/ui/ContributedPlatformTableCellJPanel.java

```diff
@@ -216,16 +251,17 @@ void update(JTable parentTable, Object value, boolean isSelected,
     } else if (selected.getParentPackage().getHelp() != null) {
       help = selected.getParentPackage().getHelp();
     }
+
     if (help != null) {
       String url = help.getOnline();
       if (url != null && !url.isEmpty()) {
-        desc += " " + format("<a href=\"{0}\">Online help</a><br/>", url);
```

Comment:
The `<br/>` went missing in the refactor (it could be a good idea to include it in both the strings to keep things cleaner)

Reply:
Good call.  Great thing to have a good code review!

Addback back for both libraries.